test/src/test/java/hudson/cli/CLIActionTest.java

```diff
@@ -255,6 +256,7 @@ public void encodingAndLocale() throws Exception {
         // -ssh mode does not pass client locale or encoding
     }
 
+    @Ignore("TODO JENKINS-46659 seems to be broken")
```

Comment:
[JENKINS-46659](https://issues.jenkins-ci.org/browse/JENKINS-46659) appears to have been the cause of the earlier failures, as well as those in #2754.